setup.py

```diff
@@ -103,4 +103,6 @@ def _pre_install(command):
     long_description=open(os.path.join(HERE, 'README.md')).read(),
     url='https://github.com/appnexus/pyrobuf',
     author='AppNexus',
+    setup_requires=['jinja2', 'cython'],
+    install_requires=['jinja2', 'cython'],
```

Comment:
Should change `'cython'` to `'cython>=0.23'`; IIRC, we are using some features of Cython that aren't present in earlier versions.